src/components/routes/Host/ListingForm/ListingInfoForm/ListingInfoForm.tsx

```diff
@@ -41,6 +42,9 @@ const ListingInfoForm = (props: FormikProps<ListingInput>): JSX.Element => {
             <Svg className="suffix" src="utils/carat-down" />
           </label>
         </SelectBoxWrapper>
+        <ErrorMessageWrapper>
+          <ErrorMessage name="homeType" />
+        </ErrorMessageWrapper>
```

Comment:
Coding in an `ErrorMessageWrapper` for each form input creates a lot of repetition. Can this be hidden behind `InputWrapper` and similar classes? (Or should we punt that until we decide to implement BeeForm?)

Reply:
ah `<ErrorMessage />` is a `Formik` component. It renders a simple string with no divs. `<ErrorMessageWrapper />` was used to tag the component and provide css.

`<ErrorMessage />` can also render its own divs. Maybe we can leverage the small api provided to clean up our implementation. https://jaredpalmer.com/formik/docs/api/errormessage. I'll keep thinking/trying different implementations throughout this week =D

Reply:
Is there an instance where <ErrorMessage /> is used with <ErrorMessageWrapper /> ?
you can write a short helper:
```
const StyledErrorMessage = (prop) => <ErrorMessageWrapper>
<ErrorMessage {...prop}>
</ErrorMessageWrapper>;
```

Reply:
updated